src/components/Common/AutoFocusInput.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <b-form-input
+    :ref="refName"
+    v-model="value"
+    v-bind="$attrs"
+    @blur="looseFocus"
+    data-auto-focus="true"
+  />
+</template>
+
+<script>
+export default {
+  name: 'AutoFocusInput',
+  props: {
+    name: {
+      type: String,
+      required: true
+    },
+    initialValue: {
+      type: String,
+      default: ''
+    }
+  },
+  computed: {
+    refName() {
+      return `auto-focus-input-${this.name}`
+    }
+  },
+  watch: {
+    value () {
+      this.$emit('input', this.value)
+    }
+  },
+  data () {
+    return {
+      value: this.initialValue,
+      isFocus: false
+    }
+  },
+  methods: {
+    // AutoFocusInput is only activated
+    enable () {
+      const tagType = document.activeElement.tagName
+
+      // If we are currently selecting another input
+      if (tagType === 'INPUT' && !document.activeElement.dataset.autoFocus) {
+        return false
+      }
+
+      return true
+    },
+    looseFocus() {
```

Comment:
Same here: should be a getter, I think.

Reply:
I didn't understand because this method only set the `isFocus` property?

Reply:
I mean
```
computed: {
  isFocus() {
    return this.refs[this.refName].isFocus
  }
}
```
And let the DOM manage the focus state. Or I'm probably missing something fundamental about the purpose of this component.

Reply:
I had to use a method because computed property are only updated on state change

Reply:
ah good point